commands/team_users_e2e_test.go

```diff
@@ -132,10 +132,10 @@ func (s *MmctlE2ETestSuite) TestTeamUserAddCmd() {
 
 		nonexistentUserEmail := "nonexistent@email"
 		err := teamUsersAddCmdF(c, &cobra.Command{}, []string{team.Id, nonexistentUserEmail})
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```

Comment:
You can check this with `s.Require().Error(err)`, which makes the test messages easier to read.